types/jquery/index.d.ts

```diff
@@ -693,7 +699,7 @@ interface JQuery<TElement extends Node = HTMLElement> {
      * @since 1.0
      * @since 1.4.3
      */
-    fadeIn(duration_easing: JQuery.Duration | string, complete?: (this: TElement) => void): this;
+    fadeIn(duration_easing: JQuery.Duration | string, complete: (this: TElement) => void): this;
```

Comment:
Why are you removing the optionality of the callbacks? This makes it so the completion callbacks become _required_ arguments.